src/Core/Components/DesignSystemProvider/FluentDesignTheme.razor.cs

```diff
@@ -8,12 +8,15 @@ public partial class FluentDesignTheme : ComponentBase
 {
     private const string JAVASCRIPT_FILE = "./_content/Microsoft.FluentUI.AspNetCore.Components/Components/DesignSystemProvider/FluentDesignTheme.razor.js";
     private DotNetObjectReference<FluentDesignTheme>? _dotNetHelper = null;
-    private string? _direction;
+    private LocalizationDirection _direction;
     private readonly JsonSerializerOptions JSON_OPTIONS = new JsonSerializerOptions
     {
         PropertyNameCaseInsensitive = true,
     };
 
+    [Inject]
+    private GlobalState GlobalState { get; set; } = default!;
```

Comment:
I think renaming is a good thing to do but it would be a breaking change so not appropriate now. 
I like `GlobalDesign`! Let's reserve that one for when we do want to do the rename. I'll update the source documentation that this is something we are going to do in the next major version.

Reply:
You could create a `GlobalDesign` class and set `Obsolete` all usages of `GlobalState` (but keeping the property). A lot of references?